extensions/sampled-rate.md

```diff
@@ -30,9 +30,9 @@ they impose additional sampling.
 
 ## Encoding
 
-### In-memory formats
+### Stand-alone event formats
```

Comment:
let's just re-do this section to talk about what the 'attribute' is - like tracing does. Drop the notion of in-memory or stand-alone